docs/src/main/paradox/stream/operators/Source-or-Flow/statefulMapConcat.md

```diff
@@ -6,7 +6,7 @@ Transform each element into zero or more elements that are individually passed d
 
 @@@ warning
 
-The `statefulMapConcat` operator has been deprecated.
+The `statefulMapConcat` operator doesn't handle upstream's completion signal since the state kept in the closure can be lost.
```

Comment:
I think I might be able to come up with a fuller description later today or tomorrow. Could we hold off on merging?

Reply:
Please change directly on this pr